src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -541,6 +543,44 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			if  (CurrentBook == null) return false;
+
+			var fileName = localPath;
+			var pos = fileName.LastIndexOfAny(new[] { '\\', '/' });
+			if (pos > -1) fileName = fileName.Substring(pos + 1);
+
+			// try to find the css file
+			var path = CurrentBook.GetFileLocator().LocateFile(fileName);
```

Comment:
This might be the last place (see later notes) that needs CurrentBook. But again, I don't think a book should be using CSS files that aren't either (a) in the book folder or (b) part of the Bloom distribution (apart from the one special case of CollectionStyles, see below). So I think it would be valid to just see if the file exists at the localPath that is produced by the href, and if not, look for it using the book-independent file locator.